tests/rbd_mirror/test_9475.py

```diff
@@ -37,12 +25,33 @@ def run(**kw):
                     node=node,
                     check_ec=False,
                 )
-            p.spawn(mirror1.benchwrite, imagespec=imagespec, io=config.get("io-total"))
+            p.spawn(
+                mirror1.benchwrite,
+                imagespec=imagespec,
+                io=config[pool_type].get("io_total"),
+            )
         time.sleep(30)
         mirror1.check_data(peercluster=mirror2, imagespec=imagespec)
-        mirror1.clean_up(peercluster=mirror2, pools=[poolname])
+        mirror1.clean_up(peercluster=mirror2, pools=[pool])
         return 0
 
     except Exception as e:
         log.exception(e)
         return 1
+
+
+def run(**kw):
+    log.info("Starting CEPH-9475")
+
```

Comment:
Please add the test case description along with the steps

Reply:
Done.